_posts/2022-05-31-introduction-to-project-panama-part-1.markdown

```diff
@@ -216,35 +226,33 @@ public static void main(String[] args) throws Throwable {
 }
 ```
 
-Note that we are allowed to cast a return value to _int_, because of the value layout of a return value defined within _printf_ **FunctionDescriptor**.
-
-In a real-life, instead of _int_ as a return value there possibly would be just a **MemorySegment** that you'd need to read
-from to get a value of a type (a struct, for instance). I'll show you more examples in the following parts!
+Note that we are allowed to cast a return type to _int_, because a return value layout carrier type of C _printf_ is _int.class_.
```

Comment:
The reason you give is "_a return value layout carrier type of C _printf_ is _int.class_._" but honestly I have no idea what this mean.